contracts/src/Allowlist.1.sol

```diff
@@ -95,4 +123,35 @@ contract AllowlistV1 is IAllowlistV1, Initializable, Administrable {
 
         emit SetAllowlistPermissions(_accounts, _permissions);
     }
+
+    /// @inheritdoc IAllowlistV1
+    function setDenyPermissions(address[] calldata _accounts, uint256[] calldata _permissions) external {
+        if (msg.sender != DenierAddress.get()) {
+            revert LibErrors.Unauthorized(msg.sender);
+        }
+
+        if (_accounts.length == 0) {
+            revert InvalidAlloweeCount();
+        }
+
+        if (_accounts.length != _permissions.length) {
+            revert MismatchedAlloweeAndStatusCount();
```

Comment:
And this one MismatchedAlloweeAndStatusCount()

Reply:
I have refactored it to `MismatchedArrayLengths`